<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="truyenltpc" failures="1" tests="1" name="testCaseHSTL.TC20_HoSoTaiLieuKyMoc_LuongKyNhay" time="238.341" errors="0" timestamp="2021-08-28T20:48:50 ICT" skipped="0">
  <testcase classname="testCaseHSTL.TC20_HoSoTaiLieuKyMoc_LuongKyNhay" name="TC20_HoSoTaiLieuKyMoc_LuongKyNhay" time="238.341">
    <failure message="expected [Yêu cầu phê duyệt &quot;[2808][VTTest] Hồ sơ tài liệu ký Mộc - Luồng ký nháy 28-08-2021 08:51&quot;] but found [Yêu cầu phê duyệt &quot;[280821][VTTest]- YCHC đổi người xử lý&quot;]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Yêu cầu phê duyệt "[2808][VTTest] Hồ sơ tài liệu ký Mộc - Luồng ký nháy 28-08-2021 08:51"] but found [Yêu cầu phê duyệt "[280821][VTTest]- YCHC đổi người xử lý"]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at pages.DanhSachViecCanXuLy.ValidateTaskYeuCauPheDuyetHopDong(DanhSachViecCanXuLy.java:288)
at testCaseHSTL.TC20_HoSoTaiLieuKyMoc_LuongKyNhay.TC20_HoSoTaiLieuKyMoc_LuongKyNhay(TC20_HoSoTaiLieuKyMoc_LuongKyNhay.java:54)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TC20_HoSoTaiLieuKyMoc_LuongKyNhay -->
  <system-out>
http://eoffice.petrolimex.com.vn/plxb12qn/vanban/SitePages/VanBanDi_Detail.aspx?IId=679&SPId=108&ListID=9710b5f7-0241-4714-8041-0f596f9ae605  </system-out>
</testsuite> <!-- testCaseHSTL.TC20_HoSoTaiLieuKyMoc_LuongKyNhay -->
